images/installer/Dockerfile.upi.ci.rhel8

```diff
@@ -84,11 +84,17 @@ RUN mkdir /output && HOME=/output && \
     ibmcloud plugin install cloud-internet-services -f && \
     cp /usr/local/bin/ibmcloud /bin/ibmcloud && \
     rm -f $HOME/.curlrc && \
-    ibmcloud version && \
-    ibmcloud plugin list
+    chmod -R g=u "$HOME/.bluemix/"
+
 
 RUN chown -R 1000:1000 /output
 USER 1000:1000
 ENV PATH /bin
 ENV HOME /output
 WORKDIR /output
+
+RUN --user 1000:1000 \
```

Comment:
Per the output in https://github.com/openshift/installer/pull/6627#discussion_r1033084167, I do not think this is right.